src/ploneintranet/api/tests/test_microblog.py

```diff
@@ -0,0 +1,25 @@
+from ploneintranet.api.testing import IntegrationTestCase
+from ploneintranet import api as pi_api
+from ploneintranet.microblog.statusupdate import StatusUpdate
+
+
+class TestMicroblog(IntegrationTestCase):
+
+    def test_create_statusupdate(self):
+        text = 'Hello this is my update'
+        update = pi_api.microblog.create_statusupdate(text)
+        self.assertIsInstance(update, StatusUpdate)
+        self.assertEqual(update.text, text)
+
```

Comment:
Could indeed switch to `api.microblog.statusupdate.get` and `api.microblog.statusupdate.create`

Anyone else agree?